### DISCUSSION TITLE
Need Help for dynamic simulation with Pandapower

### DISCUSSION THREAD

**Discussion by Adarsh-365** [CURENT/andes#480] [Q&A]
so i am trying to create dynamic simulation Load Frequency Control (LFC) scenario 
i  have created grid network in Pandapower and i want to

1)Add Load of 1000Mw due to Load  imbalance  causing frequency will fall 
2) Then i will Remove that load then frequency will maintain 

any idea how can i do it
also  i have grid network in PSS/E .raw and .dyr file in case Pandapower doesn't support

**Comment by jinningwang** [comment]
ANDES has an interface with pandapower [pandapower Interface](https://github.com/CURENT/andes/blob/master/docs/source/examples/pandapower.ipynb), and a discussion [Simple interface for PandaPower](https://github.com/CURENT/andes/discussions/265) is also available for reference.

Examples in ANDES documentation will be helpful for you to manipulate file conversion, event trigger, and finally LFC.

**Reply by Adarsh-365** [reply]
from discussion https://github.com/CURENT/andes/discussions/265#discussioncomment-2121498 

i am looking for,  system = from_pandapower(Netowrk made in pandpower). can you tell me where i can find
